3-enrich/stream-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/kinesis/sinks/NsqSink.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2013-2017 Snowplow Analytics Ltd.
+ * All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache
+ * License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.
+ *
+ * See the Apache License Version 2.0 for the specific language
+ * governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.kinesis
+package sinks
+
+// Java
+import java.nio.charset.StandardCharsets.UTF_8
+
+// NSQ
+import com.github.brainlag.nsq.NSQProducer
+
+
+/**
+  * Stdouterr Sink for Scala enrichment
+  */
+class NsqSink(config: KinesisEnrichConfig, inputType: InputType.InputType) extends ISink {
+
+  private val producer = new NSQProducer().addAddress(config.nsqHost, config.nsqdPort).start();
+  private val topicName = inputType match {
+    case InputType.Good => config.nsqGoodSinkTopicName
+    case InputType.Bad => config.nsqBadTopicName
+  }
+
+  /**
+    *
+    * @param events Sequence of enriched events and (unused) partition keys
+    * @return Whether to checkpoint
+    */
+
+  override def storeEnrichedEvents(events: List[(String, String)]): Boolean = {
+    events foreach {
+      e => producer.produce(topicName, e._1.getBytes(UTF_8))
```

Comment:
it's only one message sent to nsq instead of n: https://github.com/brainlag/JavaNSQClient/blob/master/src/main/java/com/github/brainlag/nsq/NSQCommand.java#L97-L109